aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,10 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## September 20 - October 1, 2021
+- New resources and a checklist has been added to the [Accessibility page](/foundation/accessibility).
```

Comment:
Can we link to these resources directly? Lots of text on that page. It'd be nice to have a shortcut.

Reply:
Yeah I will update the link